SuperSport/src/main/kotlin/services/ScrapperServiceImpl.kt

```diff
@@ -5,7 +5,52 @@ import models.SiteScrap
 class ScrapperServiceImpl (private val browserService: BrowserService): ScrapperService {
 
     override suspend fun scrapPage(): SiteScrap {
-        TODO("Not yet implemented")
+        val matchDetails = extractMatchDetails()
+
+        //click events
```

Comment:
avoid comments as much as possible, rather name functions in a descriptive way or the parameter names should guide the reader on whats going on. e.g. with this click tab function right, you can have the following

val events = 8
clickTab(event, "menu-item")

also use named params
instead of clickTab(event, "menu-item")

write clickTab(pos = event, className = "menu-item")

I'll explain the importance of this in the next standup meeting